Automattic/wp-calypso
# Extensions: Zoninator - Change navigation design

This PR aims to improve the navigation across settings for the Zoninator plugin based on @rickybanister's mock-ups in #16254 .

![screen shot 2017-07-19 at 21 25 38](https://user-images.githubusercontent.com/8056203/28385909-1f21e5aa-6cca-11e7-89f4-c5334c9859ff.png)
![screen shot 2017-07-19 at 21 26 29](https://user-images.githubusercontent.com/8056203/28385910-1f268ccc-6cca-11e7-8923-98a13675e627.png)
![screen shot 2017-07-19 at 21 27 04](https://user-images.githubusercontent.com/8056203/28385911-1f2e8620-6cca-11e7-9bda-d511cd0dab97.png)

# Testing

- Head over to Zoninator's plugin details and proceed to "Edit plugin settings".
- Try clicking on individual zones and "Add a zone" button and verify the pages and URLs are switching correctly.
- Verify that the back buttons on "Add a zone" and "Edit zone" pages redirects back to "Zoninator Settings".
- Verify that the back button on "Zoninator Settings" redirects to plugin details page.
